internal/wasm/store.go

```diff
@@ -430,6 +423,7 @@ func (m *ModuleInstance) resolveImports(module *Module, importedModules map[stri
 			if expected.Type != importedTable.Type {
 				err = errorInvalidImport(i, idx, fmt.Errorf("table type mismatch: %s != %s",
 					RefTypeName(expected.Type), RefTypeName(importedTable.Type)))
+				return
```

Comment:
while there, I noticed this missing `return` and we haven't added the unit test to cover this, so I backfilled it.

Reply:
oops :)